crates/walrus-sdk/src/client.rs

```diff
@@ -2298,7 +2298,7 @@ impl<T> WalrusNodeClient<T> {
 
     /// Returns a [`ClientError`] with [`ClientErrorKind::BlobIdBlocked`] if the provided blob ID is
     /// contained in the blocklist.
-    fn check_blob_id(&self, blob_id: &BlobId) -> ClientResult<()> {
+    fn check_blob_is_blocked(&self, blob_id: &BlobId) -> ClientResult<()> {
```

Comment:
Suggestion: I'd recommend making it `check_blob_in_blocklist` to be even more explicit.